### ISSUE TITLE
Keep English and Chinese translation in sync

### ISSUE THREAD

**Issue by vmihailenco** [go-redis/redis-docs#8]
@monkey92t it looks like you've improved some materials during translation. Specifically:
- zh/go-redis-pool.html and en/go-redis-debugging.html have different content
- zh/go-redis-hook.html only exists in Chinese translation
- zh/bloom-cuckoo-count-min-top-k.html is untranslated and only exists in English version

@monkey92t if you know other differences, please add them here so we can fix them later.

**Comment by monkey92t** [comment]
Yes, there are a few changes (but only temporarily):

1. zh/guide/go-redis-option.md only has Chinese documents, because in `redis/go-redis`, `Options, ClusterOption struct` already has English comments.
2. zh/go-redis-pool.html added more descriptions of the pool, and renamed it to `go-redis-pool.md`. I will port it to the English version later and keep it in sync with the English version (or Do you think otherwise?).
3. zh/go-redis-hook.html already has English comments in redis/go-redis.
4. zh/bloom-cuckoo-count-min-top-k.html I am not familiar with these modules, I will supplement the Chinese documents after I understand these modules.

 
I believe we can't do them perfectly all at once, we need to perfect them gradually. But now we can already provide some help to users, right?

**Comment by vmihailenco** [comment]
>I believe we can't do them perfectly all at once, we need to perfect them gradually. But now we can already provide some help to users, right?

Absolutely, you did a tremendous job already and I will help with keeping the docs in sync as time allows.
